pyproject.toml

```diff
@@ -54,3 +54,40 @@ all = ["cleanvision[huggingface,pytorch,azure,gcs,s3]"]
 "Bug Tracker" = "https://github.com/cleanlab/cleanvision/issues"
 "Documentation" = "https://cleanvision.readthedocs.io/"
 
+[tool.isort]
+py_version = "38"
+line_length = 88
+lines_after_imports = 2
+lines_between_sections = 1
+multi_line_output = 3
+force_grid_wrap = 0
+include_trailing_comma = "true"
+combine_as_imports = "true"
+use_parentheses = "true"
+skip = ["./docs", ".direnv"]
+skip_glob = ["*/__init__.py"]
+skip_gitignore = "true"
+sections = ["FUTURE", "STDLIB", "THIRDPARTY", "FIRSTPARTY", "LOCALFOLDER"]
+default_section = "THIRDPARTY"
+
+
+[tool.black]
+line-length = 88
```

Comment:
We set a 100 character line-length limit in the cleanlab code. I believe the same applies here.

Some of the reformatted lines need to be reverted because of this.

Reply:
CC @sanjanag

Reply:
> Some of the reformatted lines may need to be reverted because of this.

yes, I will re-run the formatting.

Reply:
I ran formatting after setting it to 100, nothing has changed. 

A majority of my changes were docstrings that were extremely long, like length>200

Reply:
I added an example below: https://github.com/cleanlab/cleanvision/pull/228#discussion_r1354954914

Reply:
My mistake, I didn't realize the formatted (docstrings) were extremely long and manually split.

Then ignore my initial comment here.